grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -163,4 +165,141 @@ public static Value of(Map<Value, Value> fields) {
         .setCollection(Collection.newBuilder().addAllElements(values).build())
         .build();
   }
+
+  public static boolean bool(Value value) {
+    checkInnerCase(value, InnerCase.BOOLEAN);
+
+    return value.getBoolean();
+  }
+
+  public static int int_(Value value) {
+    checkInnerCase(value, InnerCase.INT);
+
+    int intValue = (int) value.getInt();
+    if (intValue != value.getInt()) {
+      throw new IllegalArgumentException(
+          String.format("Valid range for int is %d to %d", Integer.MIN_VALUE, Integer.MAX_VALUE));
+    }
+    return intValue;
+  }
+
+  public static long bigint(Value value) {
+    checkInnerCase(value, InnerCase.INT);
+
+    return value.getInt();
```

Comment:
shouldn't be `value.getVarint()`?

Reply:
`.getVarint()` is for `varint` not `bigint`.